### PR TITLE
Check subcmd exit codes in rebuild-iptables script

### PR DESCRIPTION
**Pull request by JCotton1123**
### Description

Ensures rebuild-iptables does actually succeed by checking subcommand exit codes.
### Issues Resolved

https://github.com/chef-cookbooks/iptables/issues/64

### CODE REVIEW

File: templates/default/rebuild-iptables.erb [hunk 1]
```diff
@@ -130,7 +133,8 @@ end
 
 system_files = %w(/etc/debian_version /etc/redhat-release /etc/system-release)
 if system_files.any? { |file| File.exist?(file) }
-  install_rules(iptables_rules)
+  success = install_rules(iptables_rules)
+  raise "#{0}: failed to install iptables rules" unless success
```
Comment (cnmcavoy) [inline templates/default/rebuild-iptables.erb]: Should `"#{0}"` be `"#{$0}"` (note the `$`)?

Comment (JCotton1123) [inline templates/default/rebuild-iptables.erb]: Nope I believe thats the correct syntax. `$0` is a *global* variable containing the name of the executable.

Comment (JCotton1123) [inline templates/default/rebuild-iptables.erb]: Sorry. I think I had a moment of dyslexia. I read your comment backwards and looked at the wrong line. Fix submitted.

**Comment by JCotton1123** [comment]
I see that you're planning to replace this script. I am wondering if I should close this PR however it might make sense to merge it b/c 1) it may be a while before the rewrite happens, 2) the rewrite is likely going to require a major version bump and it may be a while before its widely adopted. Feel free to close this PR if appropriate.

**Pull request merged**